src/content/docs/opentelemetry/nrdot/nrdot-collector.mdx

```diff
@@ -0,0 +1,82 @@
+---
+title: New Relic Distributions of the OpenTelemetry Collector
+tags:
+  - OpenTelemetry
+  - OpenTelemetry Collector
+  - nrdot-collector-host
+  - nrdot-collector-k8s
+  - NRDOT
+metaDescription: New Relic-supported pre-built distributions of the OpenTelemetry Collector
+freshnessValidatedDate: 2024-05-14
+---
+
+The New Relic Distribution of OpenTelemetry Collector (NRDOT) is a collection of New Relic-supported pre-built distributions of the OpenTelemetry [Collector](https://opentelemetry.io/docs/collector/).
+NRDOT is designed to work seamlessly with New Relic products and services, and it is the recommended way to collect, process, and export open telemetry data for use with New Relic.
+Each distribution comes bundled with configs that are pre-configured to integrate with New Relic ingest.
+
+Rather than providing an all encompassing single distribution of the OpenTelemetry Collector, we have focused on providing distributions that are:
+1. Lightweight
+2. Limited in scope
+3. Tailored to specific New Relic experiences
+4. Pre-configured to report to New Relic's OTLP endpoint
+
+This allows us to provide a more focused and tested experience for our users, while also allowing us to iterate and improve on these distributions more quickly.
+Each distribution is tailored to a specific experience in New Relic and meant to allow users to quickly get started with OpenTelemetry and New Relic.
+Our intent is that there be minimal overlap between the distributions and that they can be used in conjunction with other collectors if need be.
+Additionally, since they are built using the [OCB](https://opentelemetry.io/docs/collector/custom-collector/), advanced users can build their own distributions if they must include additional components.
+
+<Callout variant="tip">
+  There are scenarios where you may want to deploy the collector in a less opinionated way, we recommend reading our documentation on [collector processing](/docs/opentelemetry/get-started/collector-processing/opentelemetry-collector-processing-intro) to understand how to configure the collector for your specific use case.
+</Callout>
+
+Before diving into one of our distributions we recommend familiarizing yourself with the basic concepts of the OpenTelemetry Collector in order to understand how our configurations are structured and how to customize them to your needs.
+
+## Deployments [#deployments]
```

Comment:
Let's split this out as a column in the table below to reinforce this point. I.e.:

| Distribution | Descrpition | Deployment strategy |
|---|---|---|